plugins/chat/assets/javascripts/discourse/components/chat-channel-unread-indicator.gjs

```diff
@@ -10,48 +11,41 @@ export default class ChatChannelUnreadIndicator extends Component {
 
   get showUnreadIndicator() {
     return (
-      this.args.channel.tracking.unreadCount > 0 ||
-      this.args.channel.tracking.mentionCount > 0 ||
-      this.args.channel.unreadThreadsCountSinceLastViewed > 0
+      this.args.channel.tracking.unreadCount +
+        this.args.channel.tracking.mentionCount +
+        this.args.channel.unreadThreadsCountSinceLastViewed >
+      0
     );
   }
 
-  get urgentCount() {
-    if (this.hasChannelMentions) {
-      return this.args.channel.tracking.mentionCount;
-    }
-    if (this.hasWatchedThreads) {
-      return this.args.channel.tracking.watchedThreadsUnreadCount;
-    }
-    return this.args.channel.tracking.unreadCount;
-  }
-
-  get isUrgent() {
-    if (this.onlyMentions) {
-      return this.hasChannelMentions;
-    }
+  get publicUrgentCount() {
     return (
-      this.isDirectMessage || this.hasChannelMentions || this.hasWatchedThreads
+      this.args.channel.tracking.mentionCount +
+      this.args.channel.tracking.watchedThreadsUnreadCount
     );
   }
 
-  get isDirectMessage() {
+  get directUrgentCount() {
     return (
-      this.args.channel.isDirectMessageChannel &&
-      this.args.channel.tracking.unreadCount > 0
+      this.args.channel.tracking.unreadCount +
+      this.args.channel.tracking.mentionCount +
+      this.args.channel.tracking.watchedThreadsUnreadCount
```

Comment:
Adds the mention count for DM threads to the channel list indicator on mobile/drawer